pkg/storaged/stratis-details.jsx

```diff
@@ -171,9 +180,27 @@ export function validate_pool_name(client, pool, name) {
 
 export const StratisPoolDetails = ({ client, pool }) => {
     const filesystems = client.stratis_pool_filesystems[pool.path];
+    const blockdevs = client.stratis_pool_blockdevs[pool.path] || [];
 
     const forced_options = ["x-systemd.requires=stratis-fstab-setup@" + pool.Uuid + ".service"];
 
+    function grow_blockdevs() {
+        return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-110927-1232dadb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)